### PR TITLE
Update BidGetItem.php

### PR DESCRIPTION
**Pull request by git407**
Fix error in getBidsService with FieldNames = "AuctionBids"
JsonMapper::map() requires first argument to be an object, array given.

### CODE REVIEW

File: src/services/bids/models/BidGetItem.php [hunk 1]
```diff
@@ -64,5 +64,5 @@ class BidGetItem extends Model
     /**
      * @var AuctionBidItem
```
Comment (SonicGD) [inline src/services/bids/models/BidGetItem.php:65]: I think it should be this way:
```suggestion
     * @var AuctionBidItem[]
```

File: src/services/bids/models/BidGetItem.php [hunk 2]
```diff
@@ -64,5 +64,5 @@ class BidGetItem extends Model
...
      */
-    public $AuctionBids;
-}
\ No newline at end of file
+    public $AuctionBids[];
```
Comment (SonicGD) [inline src/services/bids/models/BidGetItem.php:67]:
```suggestion
    public $AuctionBids;
```

**Comment by SonicGD** [comment]
Hello. Thanks for PR. I don't think that
```php
public $AuctionBids[];
```
is a valid syntax. I think you've meant to add `[]` into phpdoc section.

**Comment by git407** [comment]
Yes, phpdoc section! Sorry...

**Pull request merged**